packages/terra-button-group/src/orion-fusion-theme/ButtonGroup.module.scss

```diff
@@ -21,7 +22,7 @@
     --terra-button-group-button-selected-and-hover-color: #fff;
 
     --terra-button-group-button-selected-and-focus-background-color: #52585b;
-    --terra-button-group-button-selected-and-focus-border-color: #6f7477;
+    --terra-button-group-button-selected-and-focus-border-color: rgba(76, 178, 233, 0.3);
```

Comment:
why do we need change in focus color for fusion theme ?

Reply:
Reverted.